.github/workflows/tenzir.yaml

```diff
@@ -823,6 +827,9 @@ jobs:
           - name: Pipeline Manager
             target: pipeline-manager
             path: contrib/tenzir-plugins/pipeline-manager
+          - name: Packages
```

Comment:
We probably want to remove the Context and Pipeline Manager plugins from the list above, they're going to fail to build now that they're bundled.